nextflow.config

```diff
@@ -282,73 +269,95 @@ env {
     JULIA_DEPOT_PATH = "/usr/local/share/julia"
 }
 
-// Capture exit codes from upstream processes when piping
-process.shell = ['/bin/bash', '-euo', 'pipefail']
+// Set bash options
+process.shell = """\
+bash
+
+set -e # Exit if a tool returns a non-zero status/exit code
+set -u # Treat unset variables and parameters as an error
+set -o pipefail # Returns the status of the last command to exit with a non-zero status or zero if all successfully execute
+set -C # No clobber - prevent output redirection from overwriting files.
+"""
 
 // Disable process selector warnings by default. Use debug profile to enable warnings.
 nextflow.enable.configProcessNamesValidation = false
 
-def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_timeline_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_timeline_${params.trace_report_suffix}.html"
 }
 report {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_report_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_report_${params.trace_report_suffix}.html"
 }
 trace {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_trace_${trace_timestamp}.txt"
+    file    = "${params.outdir}/pipeline_info/execution_trace_${params.trace_report_suffix}.txt"
 }
 dag {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/pipeline_dag_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/pipeline_dag_${params.trace_report_suffix}.html"
 }
 
 manifest {
     name            = 'nf-core/pixelator'
-    author          = """Pixelgen Technologies AB"""
+    author          = """Pixelgen Technologies AB""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
+    contributors    = [
+        // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
+        [
+            name: 'Pixelgen Technologies AB',
+            affiliation: '',
+            email: '',
+            github: '',
+            contribution: [], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+    ]
     homePage        = 'https://github.com/nf-core/pixelator'
     description     = """Pipeline for analysis of Molecular Pixelation assays"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=23.04.0'
-    version         = '1.3.1'
+    defaultBranch   = 'master'
+    nextflowVersion = '!>=24.04.2'
+    version         = '1.4.0'
     doi             = '10.1101/2023.06.05.543770'
 }
 
-// Load modules.config for DSL2 module specific options
-includeConfig 'conf/modules.config'
```

Comment:
don't need modules.config ?

Reply:
This include was moved to the bottom of the file in the latest pipeline template: https://github.com/nf-core/pixelator/blob/ca529a0f889ef7b6f2d741e8f919d8c9e0834919/nextflow.config#L363C1-L363C36